tests/test-cases.js

```diff
@@ -1227,10 +1227,9 @@ var CUSTOM_TESTS = [
 				step: function(response, handle) {
 					updates.push(response.data[0]);
 					handle.abort();
-				},
-				complete: function() {
 					callback(updates);
```

Comment:
It looks like the code still calls complete (just glancing at the diff; haven't had a chance to pull it down) but yeah, the test should still assert that complete is being called, even for aborted tasks. In the case of an aborted parse, the meta property `aborted` should be `true`.